eclipse/openvsx
# Missing property LocalizedLanguages in vscode-language-pack-zh-hans ext

Fixes #619
Added LocalizedLanguages property

### Testing steps
- Publish [Simplified Chinese Language Pack](https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-zh-hans) to your local Open VSX instance.
- Get `http://localhost:8080/api/MS-CEINTL/vscode-language-pack-zh-hans`, the response should have `localizedLanguages` array with value `中文(简体)`.
- Do extensionquery, the response should have property `Microsoft.VisualStudio.Code.LocalizedLanguages` with value `中文(简体)`.
```
curl --location --request POST 'http://localhost:8080/vscode/gallery/extensionquery' \
--header 'Content-Type: application/json' \
--header 'Accept: application/json;api-version=3.0-preview.1' \
--data-raw '{"filters":[{"criteria":[{"filterType":7,"value":"MS-CEINTL.vscode-language-pack-zh-hans"},{"filterType":8,"value":"Microsoft.VisualStudio.Code"},{"filterType":12,"value":"4096"}],"pageNumber":1,"pageSize":50,"sortBy":4,"sortOrder":0}],"assetTypes":[],"flags":950}'
```
